main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -73,6 +74,7 @@ object Bootstrap {
     //
     // Create the project directories and files.
     //
+    newDirectoryIfAbsent(resourcesDirectory)
```

Comment:
I don't think we should require this directory. Can we just make it optional and include it if present?